ci/authn-k8s/dev/Dockerfile.inventory_no_permissions

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.3
```

Comment:
Is there a reason we're using such an old version?  Conjur uses 2.5 and the current stable version is 2.7 I believe.

Reply:
ummm not that i know of, i just took the existing Dockerfile